cmd/maya-apiserver/spc-watcher/storagepool_create.go

```diff
@@ -124,6 +124,7 @@ func newCasPool(spcGot *apis.StoragePoolClaim, reSync bool, pendingPoolCount int
 	pool.Type = spcGot.Spec.Type
 	pool.ReSync = reSync
 	pool.PendingPoolCount = pendingPoolCount
+	pool.Annotations = spcGot.Annotations
```

Comment:
Get annotations from spc and put it into CasPool object.
pool is a instance of CasPool object.